esp/src/src/DiskUsage.ts

```diff
@@ -11,7 +11,7 @@ Palette.rainbow("DiskUsage", ["green", "green", "green", "green", "green", "gree
 
 export class Summary extends FlexGrid {
 
-    private _connection = new MachineService({ baseUrl: "" });
+    private _connection = new MachineService({ baseUrl: "", timeoutSecs: 360 });
```

Comment:
This is just intended to occur on the Summary and not the Details?

Reply:
Yes - its only related to the disk usage...

Reply:
OK.